src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -523,10 +524,12 @@ function tryRemoveElement(elem) {
             }
         }
 
+        setSubtitleOffset = debounce(this._setSubtitleOffset, 500);
+
```

Comment:
@dmitrylyzo Let me know if this looks like something we want to implement to deal with this issue. 
It seems to work for me (even with secondary subtitles) when rapidly changing the offset value.

Even though it was already in the dependency list, I don't want to start importing from `lodash-es` if this is something we want to avoid for the repo in general

Reply:
> Even though it was already in the dependency list, I don't want to start importing from `lodash-es` if we are trying to avoid using lodash in the repo in general

`lodash-es` is already in use, so feel free to use it.

I would change `500` ms with `100` ms.

Reply:
Great sounds good. I will make that change now.